plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -444,6 +444,21 @@ public void testDropMissingStats()
         }
     }
 
+    @Test
+    public void testOldDateStats()
+    {
+        try (TestTable table = new TestTable(
+                getQueryRunner()::execute,
+                "test_old_date_stats",
+                "AS SELECT d_date FROM tpcds.tiny.date_dim")) {
+            assertQuery(
+                    "SHOW STATS FOR " + table.getName(),
```

Comment:
We're in the `TestDeltaLakeAnalyze` class, but i don't see ANAYZE here.
Wrong test class?

Reply:
We don't run ANALYZE, but the tests here are about stats and I didn't find a better match in another test class. There's a couple of other tests here which don't run ANALYZE as well.

Reply:
Which one?

Reply:
testDropStatsAccessControl and testDropMissingStats

Reply:
`drop_stats` procedure is counterpart to ANALYZE, that's why the both live in single test class.

Reply:
BTW since this is TPC-DS-  and behmark-friendly change, you can call ANALYZE here.
this will ensure ANALYZE has no issues on "historical dates" either.

Reply:
please add a code comment explaining that min/max are important for this column